app/assets/javascripts/controllers/container_dashboard/util/chart-data-mixin.js

```diff
@@ -0,0 +1,7 @@
+angular.module('miq.util').factory('ChartsDataMixin', [function chartDataMixinFactory () {
+  'use strict';
+  return {
+    dashboardHeatmapChartHeight:    286,
```

Comment:
@jeff-phillips-18 , I wonder if things like height should be really set in css rather than js. thoughts?